manifests/0000_70_dns-operator_02-deployment.yaml

```diff
@@ -75,15 +79,20 @@ spec:
           capabilities:
             drop:
             - ALL
+          readOnlyRootFilesystem: true
         terminationMessagePolicy: FallbackToLogsOnError
         volumeMounts:
         - mountPath: /etc/tls/private
           name: metrics-tls
           readOnly: true
+        - mountPath: /tmp
+          name: tmp-dir
```

Comment:
Does the kube-rbac-proxy container really need a writable `/tmp`?  Is it intentional that the dns-operator container and the kube-rbac-proxy container share a `/tmp` volume?

Reply:
Similar to above, more for a "just in case" rather than a specific need. And it's not needed that they share the /tmp dir, more just to keep from adding more emptyDirs than necessary. If this is of concern, we can remove it.